File: Backend/Larpex.Shared/ModelDto/TimeslotDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Larpex.Shared.ModelDto;

public class TimeslotDto
{
    public DateTime TimeslotDatetime { get; set; }

    public TimeSpan TimeslotDuration { get; set; }
}

File: Frontend/src/entities/TimeslotDto.ts
import TimeSpan from "./TimeSpan.ts";

interface TimeSlotDto {
  timeslotDatetime: string;
  timeslotDuration: TimeSpan;
}

interface TimeSlotDto2 {
  timeslotDatetime: string;
  timeslotDuration: string;
}

export type {
  TimeSlotDto,
  TimeSlotDto2
}